ui/scss/src/forms-engine.scss

```diff
@@ -114,6 +114,9 @@ div.xdsoft_datetimepicker {
 @import '../../static-assets/libs/bootstrap-sass-official/assets/stylesheets/bootstrap/dropdowns';
 @import '../../static-assets/libs/bootstrap-sass-official/assets/stylesheets/bootstrap/panels';
 
+// Utility classes
+@import '../../static-assets/libs/bootstrap-sass-official/assets/stylesheets/bootstrap/utilities';
```

Comment:
Bootstrap utilities was removed from this files, containing styles like for `.hidden` class, which was causing the labels to be displayed